src/Hl7.Fhir.Patch/IPatchDocument.cs

```diff
@@ -0,0 +1,18 @@
+/* 
+ * Copyright (c) 2020, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://github.com/FirelyTeam/fhir-net-api/blob/master/LICENSE
+ */
+
+using Hl7.Fhir.Patch.Operations;
+using System.Collections.Generic;
+
+namespace Hl7.Fhir.Patch
+{
+    public interface IPatchDocument
+    {
+        IList<Operation> GetOperations();
```

Comment:
Neither this method nor its implementation in `PatchDocument` is used. Instead only `PatchDocument.ApplyTo(ElementNode)` is used. 
Either remove the interface altogether or adjust it to the actually used public methods.